apps/fabric-website/src/components/PageHeaderLink/PageHeaderLink.tsx

```diff
@@ -44,26 +44,27 @@ export class PageHeaderLink extends React.Component<IPageHeaderLink, {}> {
   private _eventListener(event) {
     event.preventDefault();
     history.pushState({}, '', this._els.link.getAttribute('href'));
+    let navigatorUserAgent = navigator.userAgent.toLowerCase();
     let hash = this._extractAnchorLink(window.location.hash);
-    if (navigator.userAgent.toLowerCase().indexOf('firefox') > -1) {
+    if (navigatorUserAgent.indexOf('firefox') > -1) {
       hash = decodeURI(hash);
     }
     let el = document.getElementById(hash);
     let elRect = el.getBoundingClientRect();
     let bodySTop = document.body.scrollTop;
```

Comment:
just a note here... if you see weird class names like this, just make them better... we should just spell things out like `bodyScrollTop` instead of being lazy with a part of the name. (Not blaming you, just if you're in the area and spot this please don't hesitate to improve :)